docs/usage.md

```diff
@@ -135,5 +245,8 @@ See Step3: `CodeChecker store ./reports --host localhost --port 8555 --name tmux
 ```
 4. Compare your local analysis to the central one
 ```
- CodeChecker cmd diff -b tmux -n ./reports --new --host localhost --port 8555
+ CodeChecker cmd diff -b tmux_master -n ./reports --new --host codechecker.central --port 8555
 ```
+
+*Note:* In upcoming CodeChecker versions it will be possible to generate HTML report files for new findings
```

Comment:
Do we want to introduce this text and have it forever in the codebase. This is only a planned feature, I don't believe it belongs into the example-driven user guide.